src/css/flags/controls-disabled.less

```diff
@@ -1,6 +1,7 @@
 .jw-flag-controls-disabled {
-    .jw-controls {
+    .jw-controls, .jw-nextup-container {
         visibility: hidden;
+        opacity: 0;
```

Comment:
Make sure only visibility: hidden is set for the problem cases:

".jw-flag-controls-disabled" makes the controls container hidden
but this is overriden by visible changes in 
".jw-nextup-container-visible" and ".jw-nextup-sticky .jw-nextup-close"